draft-yasskin-wpack-bundled-exchanges.md

```diff
@@ -336,13 +370,16 @@ the parser MUST do the following:
       * header list is `headers`, and
       * client is null.
 
-   1. Let `streamOffset` be `sectionsStart +
-      sectionOffsets["responses"].offset + offset`. That is, offsets in the
-      index are relative to the start of the "responses" section.
-   1. If `offset + length` is greater than
+   1. Let `streamOffset` be `sectionOffsets["responses"].offset +
+      currentOffset`. That is, offsets in the index are relative to the start of
+      the "responses" section.
```

Comment:
Should we remove the sentence "That is, offsets in the index are relative to the start of the "responses" section."?

Reply:
Whoops, done.